workflow/exp/baselines/exp.conus12km_3denvar_b001

```diff
@@ -15,6 +15,7 @@ export DATAROOT=${OPSROOT}/exp/${EXP_NAME}/stmp    # task workdirs ($DATA) which
 export ZETA_LEVELS="L60.txt"
 #
 export YAML_GEN_METHOD=2
+export QROCOTO_OPTION=2
```

Comment:
Is it possible to reuse `YAML_GEN_METHOD` if you want to go this route?

Reply:
It is possible. I was just thinking that a user might want to try one way or another regardless of the YAML_GEN_METHOD. Do you still want me to do it that way?

Reply:
They are all still available under `workflow/ush` if one wants to try any of the scripts. 
So, I think reusing"YAML_GEN_METHOD" will be fine. Thanks!

Reply:
Sounds good to me. I just made the change. I also think all my linter problems have been fixed.